include/mathfu/vector.h

```diff
@@ -410,6 +410,25 @@ class Vector {
     MATHFU_VECTOR_OPERATION(vector->data[i] = data_[i]);
   }
 
+  /// @brief Compare 2 Vectors of the same size
+  ///
+  /// @return True if the 2 vectors contains the same value
```

Comment:
I'm not sure about that since users will probably want to compare against their own magic epsilon values.  I was mostly talking about the docs for users not versed in the pitfalls of floating point :)